src/app/logic/questionsGenerator/QuestionsGenerator.js

```diff
@@ -0,0 +1,104 @@
+import { async } from "regenerator-runtime";
+import { GameModes } from "../../Consts";
+import { getPersonById, getPersonImageBlobById } from "../../apiconnection/PeopleClient";
+import { getStarshipById, getStarshipImageBlobById } from "../../apiconnection/StarshipsClient";
+import { getVehicleById, getVehicleImageBlobById } from "../../apiconnection/VehiclesClient";
+
+
+async function generateQuestions(gameMode) {
+    let answers;
+    let rightAnswer;
+    let imageBlob;
+    if (gameMode === GameModes.PEOPLE) {
+        const allowedIds = [];
+        for (let i = 1; i < 83; i++) {
+            if (i === 17) continue;
```

Comment:
I would define what are numbers 83 and 17 as a consts

Reply:
Done